src/api/files/service.js

```diff
@@ -57,8 +57,13 @@ export async function ingestFile(uploadPayload) {
  * Confirms a file exists in S3 by throwing Boom.badRequest if not.
  * @param {FileUploadStatus} fileUploadStatus
  * @param {Error} errorToThrow
+ * @param {boolean} logAsError - whether to log the error
```

Comment:
Mind sticking optional square brackets on?

It's figuring it out by the default `logAsError = true` anyway

```suggestion
 * @param {boolean} [logAsError] - whether to log the error
```